src/index.ts

```diff
@@ -127,7 +180,8 @@ function getEmit(
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // Remove original dependency on resourcePath and replace with rawFilePath
```

Comment:
Could you provide some explanation on what is happening here?  This replaces the simple `loader.clearDependencies()` with something that will clear dependencies and then perhaps add dependencies again.  I'm not clear on what `loader.resourcePath` is and whether we always want to follow this path.  

Some details of what this should achieve would be helpful.  Also, we still have the `addDependency` call after `removeDependencies` - will this result in double adds?